packaging/test_rule_engine_plugin_metadata_guard.py

```diff
@@ -287,6 +287,28 @@ def test_plugin_cannot_be_bypassed_with_addw__issue_40(self):
             # Clean up.
             self.rods.assert_icommand(['imeta', 'rm', '-C', root_coll, self.metadata_guard_attribute_name(), json_config])
 
+    def test_plugin_cannot_be_bypassed_via_imeta_mod__issue_58(self):
+        config = IrodsConfig()
+
+        # Set JSON configuration for the root collection.
+        root_coll = os.path.join('/', self.admin.zone_name)
+        json_config = json.dumps({
+            'prefixes': ['irods::'],
+            'admin_only': True
+        })
+        self.rods.assert_icommand(['imeta', 'set', '-C', root_coll, self.metadata_guard_attribute_name(), json_config])
+
+        try:
+            with lib.file_backed_up(config.server_config_path):
+                self.enable_rule_engine_plugin(config)
+
+                self.user.assert_icommand(['imeta', 'add', '-C', self.user.session_collection, 'unprotected::issue58', 'v'])
+                self.user.assert_icommand(['imeta', 'mod', '-C', self.user.session_collection, 'unprotected::tomato', 'v', 'n:irods::tomato'], 'STDERR', ['CAT_INSUFFICIENT_PRIVILEGE_LEVEL'])
```

Comment:
i think we want to make sure it doesn't update the existing one?

"unprotected::tomato" -> "unprotected::issue58" ?

or perhaps more cleanly...

`'unprotected::issue58', 'v', 'n:irods::issue58'`

Reply:
@FifthPotato What did you determine? Was the original code a copy-paste issue?

Reply:
Yep, made a last-minute change that I didn't propagate correctly. Just changed it to what Terrell recommended.